corehq/apps/domain/templates/domain/manage_releases.html

```diff
@@ -0,0 +1,64 @@
+{% extends "hqwebapp/base_section.html" %}
+{% load hq_shared_tags %}
+{% load crispy_forms_tags %}
+{% load i18n %}
+
+{% requirejs_main 'app_manager/js/manage_releases' %}
+
+{% block page_content %}
+    {% initial_page_data 'enabled_app_releases' enabled_app_releases %}
+    {% registerurl 'deactivate_release_restriction' domain '---'%}
+    {% registerurl 'activate_release_restriction' domain '---'%}
+    <div class="row">
+        <div class="col-sm-12">
+            <div class="panel panel-modern-gray panel-form-only">
+                <div class="panel-body">
+                    <form class="form-horizontal disable-on-submit" id="manage-app-releases" action=""
+                          method='post'>
+                        {% crispy manage_releases_form %}
+                    </form>
+                {% if enabled_app_releases %}
+                    <table id="managed-releases" class="table">
+                        <thead>
+                            <tr>
+                                <th>{% trans 'Application' %}</th>
+                                <th>{% trans 'Location' %}</th>
+                                <th>{% trans 'Version' %}</th>
+                                <th>{% trans 'Last Activity' %}</th>
+                                <th>{% trans 'Action' %}</th>
+                            </tr>
+                        </thead>
+                        <tbody data-bind="foreach: enabledAppReleases, visible: enabledAppReleases">
```

Comment:
Should `visible: enabledAppReleases` go on the table tag? I don't think an empty tbody is visible anyway.

Reply:
was just thinking of letting the user know that there is an empty table so we searched but found nothing.
You think a better message would be "No entries found"

Reply:
Yeah, a message would be nice, makes it more obvious there wasn't an error.